web/app/js/components/TopModule.jsx

```diff
@@ -97,7 +98,6 @@ class TopModule extends React.Component {
 
   onWebsocketRecv = e => {
     this.indexTapResult(e.data);
-    this.props.updateNeighbors(e.data);
```

Comment:
It appears this wasn't doing anything (since not sending anything as a secong arg) so I removed it.

Reply:
good catch!